objects/drawable.c

```diff
@@ -225,6 +237,7 @@ drawable_class_setup(lua_State *L)
     static const struct luaL_Reg drawable_meta[] =
     {
         LUA_OBJECT_META(drawable)
+        { "__gc", luaA_collect },
```

Comment:
This causes leaks, because the "real" __gc handler isn't run. Also, I don't like the idea, but I guess you already know...